modules/installation-rhv-specifying-assets-directory.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-specifying-assets-directory_{context}"]
+= Specifying the assets directory
+
+Here, you create an `ASSETS_DIR` environment variable for this installation process to use.
+
+The installation program uses this variable to create the directory where it stores various files. Later, the installation process reuses this variable to locate those files.
```

Comment:
This should be part of the step, not the intro. The step isn't clear enough.

Reply:
Moved it to a note just below the step.